### PR TITLE
Bootstrap vue

### PR DESCRIPTION
**Pull request by cornelia-schulz**
Convert to Bootstrap-Vue to avoid CDN calls.

### CODE REVIEW

File: components/ProductDetails/ProductDetailsItinerary.vue [hunk 1]
```diff
@@ -80,6 +80,10 @@ h2, h3, .expand {
     color: #103A5B;
 }
 
+#itinerary h3 {
+    font-size: 14px;
```
Comment (AriesNg) [inline components/ProductDetails/ProductDetailsItinerary.vue:84]: Please check if we need responsive font size
https://www.w3schools.com/howto/howto_css_responsive_text.asp

File: components/ProductDetails/ProductDetailsOptionalTours.vue [hunk 2]
```diff
@@ -113,12 +112,15 @@ export default {
         },
         markButtonActive: function(content) {
             EventBus.$emit('markButtonActive', content)
+        },
+        closeModal: function(event) {
+            console.log(event.target.parent)
```
Comment (AriesNg) [inline components/ProductDetails/ProductDetailsOptionalTours.vue]: Do we need to do anything to closeModal?

File: components/ProductDetails/ProductDetailsOptionalTours.vue [hunk 3]
```diff
@@ -146,14 +148,47 @@ h3 {
...
+
+.optional-tour .modal-body,
+.extensions .modal-body {
+    padding-top: 0px !important;
+    padding-bottom: 0px !important;
+}
+
+.optional-tour .modal-body img,
+.extensions .modal-body img {
     height: 220px;
```
Comment (AriesNg) [inline components/ProductDetails/ProductDetailsOptionalTours.vue]: Would this height work in mobile?

File: components/Shared/HeaderAgent.vue [hunk 4]
```diff
@@ -66,45 +62,49 @@ export default {
...
 }
 
-.agent-dropdown-menu a {
+.agent-dropdown .dropdown-menu a {
     color: #1B75BB;
 }
 
-.agent-dropdown-menu p {
+.agent-dropdown .dropdown-menu p {
     line-height: 1.5em;
```
Comment (AriesNg) [inline components/Shared/HeaderAgent.vue]: should use rem instead of em for easier control

**Review by AriesNg — APPROVED** [review]
Please update based on my comments then you may merge.

File: components/ProductDetails/ProductDetailsOptionalTours.vue (hunk 2 above)
Comment (cornelia-schulz) [reply components/ProductDetails/ProductDetailsOptionalTours.vue]: No, this is redundant code, I used the close button provided by Bootstrap-Vue. I will remove the close function.

File: components/ProductDetails/ProductDetailsOptionalTours.vue (hunk 3 above)
Comment (cornelia-schulz) [reply components/ProductDetails/ProductDetailsOptionalTours.vue]: I have added this restriction to the media queries, so it only effects large screens.

File: components/Shared/HeaderAgent.vue (hunk 4 above)
Comment (cornelia-schulz) [reply components/Shared/HeaderAgent.vue]: updated.

File: components/ProductDetails/ProductDetailsItinerary.vue (hunk 1 above)
Comment (cornelia-schulz) [reply components/ProductDetails/ProductDetailsItinerary.vue:84]: I will check with the designers.

**Pull request merged**